src/test/java/com/puppycrawl/tools/checkstyle/gui/MainFrameModelTest.java

```diff
@@ -183,4 +187,30 @@ private void verifyCorrectTestDataInFrameModel() throws IOException {
         assertNotNull(model.getParseTreeTableModel(), "ParseTree table model should not be null");
     }
 
+    @Test
+    public void testShouldAcceptFile() throws IOException {
```

Comment:
I feel that each assertion should have it's own test method that describes our assertion. The method name `testShouldAcceptFile` doesn't accurately describe each assertion here.

Reply:
Done. The whole test class should be rewritten, but this is another story: #9142

Reply:
Resolved.